science/opencascade/Portfile

```diff
@@ -16,7 +16,7 @@ maintainers                 {mcalhoun @MarcusCalhoun-Lopez} openmaintainer
 description                 Development platform for CAD/CAE, \
                             3D surface/solid modeling and data exchange
 long_description            ${description}
-homepage                    https://www.opencascade.com/content/open-source-core-technology
+homepage                    https://github.com/Open-Cascade-SAS/OCCT
```

Comment:
This line can probably be dropped because this is being implicitly set by `github.setup`.

Reply:
Line dropped.  Did not know it worked like that.  Thanks.